### ISSUE TITLE
This is awesome!

### ISSUE THREAD

**Issue by BethMatta-NOAA** [MargaretSiple-NOAA/googledrive-to-rmd-example#3]
Thank you so much for sharing this, Megsie. It tried it and it works great! I have been struggling with how to develop a workflow that will accommodate varying experience levels among collaborators. This is a really big help!

**Comment by MargaretSiple-NOAA** [comment]
Awesome! I'm glad it's helpful. I have really enjoyed using this type of workflow myself. As you probably saw in Slack (or some other Openscapes forum... sometimes I lose track of where things are), the [Trackdown package](https://cran.r-project.org/web/packages/trackdown/vignettes/trackdown-workflow.html) is a slicker way of doing some of this stuff, but for my own purposes a sort of "one-directional" workflow works fine for my purposes. Anyway, that package is available if you find yourself wanting to go through iterations of the final product with coauthors ♻

**Comment by BethMatta-NOAA** [comment]
Thanks, Megsie! I'll check that package out.

**Comment by MargaretSiple-NOAA** [comment]
Yeah I haven't tried it out myself. I have heard mixed reviews, but just wanted to make sure you know about other options so you can see what works best for your purposes.
